src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -407,6 +407,50 @@ class ApplicationImpl : public virtual Application,
 
   void SwapMobileMessageQueue(MobileMessageQueue& mobile_messages) OVERRIDE;
 
+  /**
+   * @brief Get map of pending button subscription requests correlation ids
+   * to button names
+   * @return pending button subscriptions map
+   */
```

Comment:
@mked-luxoft Remove descriptions from impl class

Reply:
@AByzhynar fixed in 2b001c6014447a350d41a2bcd1e834fd2c83057a